packages/subgraph/templates/subgraph.template.yaml

```diff
@@ -73,6 +73,8 @@ templates:
       abis:
         - name: Poll
           file: ./node_modules/maci-platform-contracts/build/artifacts/contracts/maci/Poll.sol/Poll.json
+        - name: Registry
+          file: ./node_modules/maci-platform-contracts/build/artifacts/contracts/registry/BaseRegistry.sol/BaseRegistry.json
```

Comment:
It should not be BaseRegistry because it's abstract contract. It should be EAS or Simple.

Reply:
but this is just a template? I have a question, should we suggest the user to made their own subgraph.yaml instead of changing this file? since different user would try to deploy different kind of registry, we just need to remind them to replace this part and keep sync with the deploy-config.json file

Reply:
This template cannot be reused as-is. Users can update it whatever they want.

Reply:
and I have a question, since I'm not fully understand how does this template work, how many `BaseRegistry` should be updated to the `Simple` or `EAS` one? or just the one inside the `Poll` template mapping?

Reply:
Ok. so it doesn't matter for now. We just allow to parse the events here. So, let's keep only base one.